example/src/NitroAssetImageTab.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import { FlatList, StyleSheet, Text, View } from 'react-native';
+import { Image, loadImageFromAssetAsync, NitroImage } from 'react-native-nitro-image';
+import { CameraRoll } from '@react-native-camera-roll/camera-roll';
+
+
+function useAssetImage(
+  url: string,
+): Image | undefined {
+  const [image, setImage] = useState<Image | undefined>(undefined);
+
+  useEffect(() => {
+    const load = async () => {
+      try {
+        const i = await loadImageFromAssetAsync(url.replace(/^ph:\/\//, ''));
+        setImage(i);
+      } catch (error) {
+        console.error(`Failed to load image from "${url}"!`, error);
+        setImage(undefined);
+      }
+    };
+    load();
+    // `options` is missing from dependencies since it's a reference type that will be constructed each render.
```

Comment:
Isn't there a react-hooks rule that will be triggered here? That should be lint-ignored I guess. Not sure if I even have ESLint set up lol